### DISCUSSION TITLE
Add GoodDollar to GitCoin's Passport

### DISCUSSION THREAD

**Discussion by KazanderDad** [gitcoinco/passport#172] [Ideas]
Hi dPopp team,

The team over here at GoodDollar would like to get in touch with you. We like your Passport initiative, and would like to be part of it. Our G$ Wallet works as a form of self sovereign identity, and we recently built functionality to sign in with "GoodDollar" so users can easily register to services that work with G$s. Each one of our 400k+ claimers have been individually verified as a unique human being through face verification with 1:125M false positives and less than 1% false negatives. As such, our solution could act as a very good spam/bot prevention, much more useful than Google/Facebook/Twitter for any service that requires unique-human registration :-)

Please reach out for a quick chat: https://calendly.com/gooddollar-noak/30min

**Comment by korovka22** [comment]
Great idea, to start stoping those bots!

**Comment by raymonredington** [comment]
hmmmmmmmmmmmm
